common/src/main/java/com/vexsoftware/votifier/util/standalone/StandaloneVotifierPlugin.java

```diff
@@ -77,6 +59,16 @@ public String getVersion() {
         return "2.3.5";
```

Comment:
WOW this hasn't been updated in awhile.

Do we need this?

Reply:
So we really do need this, but maybe it should be in the interest of security to simply respond "2"

Reply:
We don't really need this, I can change it to 2.